docs/en/observability/logs-stream.asciidoc

```diff
@@ -31,60 +31,42 @@ include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widge
+The following is an example of a simple standalone {agent} configuration:
 
-. Go to the {kib} home page and click *Add integrations*:
-+
---
-[role="screenshot"]
-image::images/add-integrations.png[{kib} home page]
---
-
-. In the search bar, search for *custom* and select *Custom Logs*.
-. Click *Add Custom Logs* in the upper-right corner:
-+
---
-[role="screenshot"]
-image::images/add-custom-logs.png[add custom logs button location]
---
-
-. Click *Install {agent}*.
-. Click *standalone mode*:
-+
---
-[role="screenshot"]
-image::images/standalone-link.png[link to running agents in standalone mode]
---
-. Add the path to your log file in the *Log file path* field.
-. Set the name for your dataset data stream in the *Dataset name* field. The dataset name describes the data ingested and its structure. This field can contain anything that signifies the source of the data. The default value is `generic`.
-. Click *Advanced options* to open the *Integration settings*.
-+
---
-[role="screenshot"]
-image::images/custom-log-advanced-options.png[open integration settings]
---
-. Set the name for your namespace data stream in the *Namespace* field. The namespace is useful for grouping data such as an environment (dev, prod, or qa), a team, or a strategic business unit. Using a namespace makes it easier to search for data from a given source by using a matching pattern. The default value is `default`.
-. Click *Save and continue*.
-. Click *Copy to clipboard*, and paste the configuration in the `elastic-agent.yml` file located on your host where you extracted the {agent} installation package.
-. We recommend using an API key to grant the agent access to {es}. To create an API key for your agent, see {fleet-guide}/grant-access-to-elasticsearch.html#create-api-key-standalone-agent[Create API keys for standalone agents].
-. Replace `username: '${ES_USERNAME}'` and `password: '${ES_PASSWORD}'` with `api_key:` in the `elastic-agent.yml` file, and add the API key you created in the previous step. For example:
-+
 [source,yaml]
 ----
-[...]
 outputs:
   default:
     type: elasticsearch
-    hosts:
-      - 'https://da4e3a6298c14a6683e6064ebfve9ace.us-central1.gcp.cloud.es.io:443'
-    api_key: _Nj4oH0aWZVGqM7MGop8:349p_U1ERHyIc4Nm8_AYkw
-[...]
+    hosts: '<your-elasticsearch-url>:<port>'
+    api_key: 'your-api-key'
+inputs:
+  - id: your-log-id
+    type: logfile
+    data_stream:
+      namespace: default
+    streams:
+      - id: your-log-stream-id
+        data_stream:
+          dataset: generic
+        paths:
+          - /var/log/your-logs.log
 ----
+
+To configure your {agent}, set the values for the fields in the example configuration:
+
+. `hosts` – Your {es} URL and the port.
+. `api-key` – An API key to grant the agent access to {es}. To create an API key for your agent, see {fleet-guide}/grant-access-to-elasticsearch.html#create-api-key-standalone-agent[Create API keys for standalone agents].
```

Comment:
The link to the API key creation is ok for now. But I think this could become a stumbling point. We should improve this on the UX side ( @grabowskit ). Ideally we could give the user something they can just copy paste into Dev Tools or Click a single button to the get API key (we have this planned) instead of having to go to the right place, put the right json in and then select the encoding.